showcase/src/app/getting-started/getting-started-component/starter-structure.ts

```diff
@@ -0,0 +1,125 @@
+|   |   |   environment.ts              # development configuration
+|   |   \\   model.ts                    # interface defining the variables
+|   |
+|   +---styles                          # application-specific styles
+|   |   \\
+|   |
+|   |   custom-typings.d.ts             # add your own type definitions that can't be found in the registry in this file
+|   |   hmr.ts                          # Hot Module Replacement logic
+|   |   index.html                      # the main HTML page that is served when someone visits this site
+|   |   main.browser.ts                 # Angular bootstrapping
+|   |   polyfills.browser.ts            # this file includes polyfills needed by Angular and should be loaded before the app
+|   |   stark-app-config.json           # Stark configuration
+|   \\   stark-app-metadata.json         # Application metadata like: name, description, version...
+|
+|   .dockerignore                       # files and directories to be excluded by the Docker build
+|   .gitignore                          # files and directories to be excluded by git
+|   .prettierignore                     # files and directories to be excluded by prettier
+|   .prettierrc.js                      # prettier configuration file
+|   .stylelintrc                        # stylelint configuration file
+|   .travis.yml                         # YAML file to customize the Travis build (https://travis-ci.org/)
+|   angular.json                        # Angular configuration file
+|   base.spec.ts                        # Initializes the test environment
+|   Dockerfile                          # the commands that will be executed by the Docker Build command
+|   karma.conf.ci.js                    # Karma configuration file for Continuous Integration
+|   karma.conf.js                       # Karma configuration file
+|   package.json                        #
+|   protractor.conf.js                  # protractor configuration file
+|   README.md                           # this document
+|   tsconfig.app.json                   # typescript configuration for the application, extends tsconfig.json
+|   tsconfig.e2e.json                   # typescript configuration for the e2e tests, extends tsconfig.json
+|   tsconfig.json                       # typescript configuration, extends stark-build/tsconfig.json
+|   tsconfig.spec.json                  # typescript configuration for the Karma tests, extends tsconfig.json
+|   tslint.json                         # tslint configuration file
+\\   webpack.config.js                   # exports the webpack config file from config folder according to the current environment`;
+
+export const stylesheetImport: string = `|
+|   +---styles                          # application-specific styles
+|   |   \
+|   |
+|   |   custom-typings.d.ts             # add your own type definitions that can't be found in the registry in this file
+|   |   hmr.ts                          # Hot Module Replacement logic
```

Comment:
In the PrettyPrint component, the text on the right in this code block is not well aligned, could you align them?

![unaligned-pretty-print-1](https://user-images.githubusercontent.com/5698318/48055157-38024780-e1af-11e8-9f2d-b3fa7a2280b2.PNG)